traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -178,71 +178,71 @@
           "_comments": "This is the text used to configure delivery service field titles/descriptions",
           "active": {
             "title": "Active",
-            "desc": "When this is set to false Traffic Router will not serve DNS or HTTP responses for this delivery service."
+            "desc": "Whether or not this delivery service is active on the CDN and is capable of traffic."
           },
           "cacheurl": {
             "title": "Cache URL expression",
-            "desc": "Cache URL rule to apply to this delivery service.<br><br>See <a href=\"https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/cacheurl.en.html\" target=\"_blank\">ATS documentation on cacheurl...</a>"
+            "desc": "Allows you to manipulate the cache key of the incoming requests. Normally, the cache key is the origin domain. This can be changed so that multiple services can share a cache key, can also be used to preserve cached content if service origin is changed.<br><b>Note:<b> Only valid in ATS 6. X and earlier. Must be empty if using ATS 7. X and / or the <a href = \"https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/cachekey.en.html\" target=\"_blank\">cachekey plugin.</a><br><br>See <a href = \"https://docs.trafficserver.apache.org/en/6.2.x/admin-guide/plugins/cacheurl.en.html\" target=\"_blank\">ATS documentation on cacheurl </a>"
           },
           "ccrDnsTtl": {
             "title": "Delivery Service DNS TTL",
-            "desc": "The Time To Live on the DNS record for the Traffic Router A and AAAA records."
+            "desc": "(For DNS routed delivery services only) The Time To Live on the DNS record for the Traffic Router A and AAAA records. Default value is 30 seconds. Setting too high or too low will result in poor caching performance."
```

Comment:
I don't think you need the 

(For DNS routed delivery services only) 

because this field only shows up for DNS delivery services so it is redundant.

Also, the default is configurable so by saying "Default value is 30 seconds" this will be inaccurate if somebody changes the default.